### PR TITLE
docs: extend .env usage description

### PR DESCRIPTION
**Pull request by uniqueg**
As discussed.

### CODE REVIEW

File: .sample.env [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# example .env.local file; replace values with your own
+NOTION_TOKEN=secret_rrm123ABCabc456DEFdef789GHIghi012JKLjkl345M
+GA_MEASUREMENT_ID=G-123ABC45DE
```
Comment (git-anurag-hub) [inline .sample.env:3]: I think we need to remove the secrets!

File: README.md [hunk 2]
```diff
@@ -61,16 +61,25 @@ in your account, note down the "Internal Integration Token" and [link the
...
 
+For Google Analytics support, you will also need to create a Google Analytics
+account, property and data stream, and note down the measurement ("G-") ID as
+described [here][res-google-analytics-docs] (for setting up data collection,
+follow the instructions to 'Find your "G-" ID' at the bottom of the page).
+
 Create a file `.env.local` in the repository's root directory with the
-following command (don't forget to replace `YOUR_TOKEN` with your internal
-integration token):
+following command (don't forget to replace the values with your own ones):
```
Comment (git-anurag-hub) [inline README.md:70]: This can be marked optional!

File: .sample.env (hunk 1 above)
Comment (uniqueg) [reply .sample.env:3]: They are not real secrets

File: README.md (hunk 2 above)
Comment (uniqueg) [reply README.md:70]: But is it build in a way that it's optional? Nothing will break if it's not set?

Comment (git-anurag-hub) [reply README.md:70]: It is not set right now. Tanya PR is yet to be merged. We should make it optional as I think have google analytics setup is truly a hassle & is not needed for the developed working on project.

File: .sample.env (hunk 1 above)
Comment (uniqueg) [reply .sample.env]: I have put Xs to clarify that these are not real secrets

File: README.md (hunk 2 above)
Comment (uniqueg) [reply README.md]: Ah okay, I misunderstood it. Will remove all mentions from Google Analytics until it is merged then.

**Review by git-anurag-hub — APPROVED** [review]

**Comment by git-anurag-hub** [comment]
LGTM @uniqueg!

**Review by tanya17-05 — APPROVED** [review]
LGTM

**Pull request merged**